### ISSUE TITLE
Typescript support for PublishRequest of web type

### ISSUE THREAD

**Issue by iamsiddhant05** [pusher/push-notifications-node#42]
I was going through the typescript definitions and cannot find `PublishRequest` for `web`, it only has for APN and FCM ?

```
type PublishRequest =
  | PublishRequestWithApns
  | PublishRequestWithFcm
  | PublishRequestWithApnsAndFcm;
```

Can you add it or i could open a PR, but i need to know how much time will it be before you release?

**Comment by jonathanlloyd** [comment]
Thanks for raising this!
I've got a PR open & will have a release out asap

**Comment by jonathanlloyd** [comment]
@iamsiddhant05 The fix for this is available in v1.2.1
